src/validation/tests/snapshots/rusty__validation__tests__variable_validation_tests__overflow_global__.snap.new

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
snap needs insta review

Reply:
accidentally slipped in from a different branch, I deleted it